package.json

```diff
@@ -40,7 +40,7 @@
     "express": "^4.18.2",
     "multer": "^1.4.5-lts.1",
     "npm-run-all": "^4.1.5",
-    "pluralistic": "github:gitcoinco/pluralistic.js#431f4fe2d429f65f2de03bdfcc66bba451da2ba7",
+    "pluralistic": "github:gitcoinco/pluralistic.js#8f7c85f1f0ddbe91ed849e9f21a5a80c0f05916a",
```

Comment:
we need to change it to the latest where we fix a bug

Reply:
done